README.md

```diff
@@ -38,12 +38,14 @@ An RFC can be merged when it receives an absolute majority of approvals from [Ma
 
 You can also ask for help with the RFC process on Slack at [#ref-help](https://govau-guides.slack.com/archives/C02B0046A5T).
 
-An RFC can be merged once it has an absolute majority of PR approvals of all maintainers.
-
-Make sure you check the [list of maintainers](https://github.com/orgs/designsystemau/teams/maintainers), since the number of maintainers may vary.
+An RFC can be merged once it has 5 PR approvals from [Maintainers](https://github.com/orgs/designsystemau/teams/maintainers).
```

Comment:
I think making this a fixed number means that only the fastest will be heard and seeing that we don't know yet how many people are in the TSC this could mean a single entity with enough members could just push through changes without the broader community involved. I would try with absolute majority first and then change a bit later once we've done one or two.

Reply:
Restored